runtime/vm/createramclass.cpp

```diff
@@ -2844,7 +2899,11 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 		if (elementClass == NULL) {
 			ramClass = hashClassTableAt(classLoader, J9UTF8_DATA(className), J9UTF8_LENGTH(className));
 		} else {
-			ramClass = elementClass->arrayClass;
+			ramClass = createdArrayClass(elementClass,
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+				*valueTypeFlags
+#endif /* defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) */
```

Comment:
The `,` needs to be on the same line as `*valueTypeFlags`, not outside of the `if` directive.